### ISSUE TITLE
Module not found: Package path ./lib/src/react is not exported from package

### ISSUE THREAD

**Issue by aeneaswiener** [thoughtspot/visual-embed-sdk#776]
I'm attempting to use AppEmbed in a NextJS app and getting the following error

```
Failed to compile
./components/thoughtspot/fullTsApp.tsx:8:0
Module not found: Package path ./lib/src/react is not exported from package /Users/aw/rs/frontend/node_modules/@thoughtspot/visual-embed-sdk (see exports field in /Users/aw/rs/frontend/node_modules/@thoughtspot/visual-embed-sdk/package.json)
   6 |   AuthType
   7 | } from "@thoughtspot/visual-embed-sdk";
>  8 | import { AppEmbed } from "@thoughtspot/visual-embed-sdk/lib/src/react"; 
   9 | 
  10 | init({
  11 |     thoughtSpotHost:

https://nextjs.org/docs/messages/module-not-found

```

Do you know what might be causing this?

**Comment by ashubham** [comment]
@aeneaswiener Whats the version of nextjs you are using ? 

Did you try:
`import { AppEmbed } from "@thoughtspot/visual-embed-sdk/react"; `

**Comment by aeneaswiener** [comment]
using next 12.2.5

when i use

```
import { AppEmbed } from "@thoughtspot/visual-embed-sdk/react"; 
```
i get

```
Server Error
SyntaxError: Cannot use import statement outside a module

This error happened while generating the page. Any console logs will be displayed in the terminal window.
```

**Comment by ashubham** [comment]
Thanks for that detail @aeneaswiener 

Seems this is known issue https://github.com/vercel/next.js/issues/25423#issuecomment-847422832
The thread offers some workarounds as well.

FWIW, I just released version `1.17.0-alpha.4` of the `@thoughtspot/visual-embed-sdk` package where we are including the cjs definitions as well. Do you mind giving that a try ?

**Comment by aeneaswiener** [comment]
i updated to 1.17.0-alpha.4 that by itself didn't fix it

however the Option 1 workaround from your linked issue https://github.com/vercel/next.js/issues/25423#issuecomment-847422832 worked for me, so closing this issue
